pkg/joinserver/grpc_deviceregistry.go

```diff
@@ -72,7 +73,10 @@ func (srv jsEndDeviceRegistryServer) Get(ctx context.Context, req *ttnpb.GetEndD
 		dev.RootKeys = &ttnpb.RootKeys{
 			RootKeyID: rootKeysEnc.GetRootKeyID(),
 		}
-		cs := srv.JS.GetPeer(ctx, ttnpb.ClusterRole_CRYPTO_SERVER, dev.EndDeviceIdentifiers)
+		var cc *grpc.ClientConn
+		if cs, _ := srv.JS.GetPeer(ctx, ttnpb.ClusterRole_CRYPTO_SERVER, dev.EndDeviceIdentifiers); cs != nil {
```

Comment:
error handling missing

Reply:
CS is optional

Reply:
Still - it has to at least be logged as debug

Reply:
CS unavailability is normal flow, especially in open source.

As mentioned below, if we log anything, it should be if `!errors.IsNotFound(err)` because not-found is as much happy flow as found